tests/cases/fourslash/codeFixClassPropertyInitializationWithDefiniteAssignmentAssertions.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path='fourslash.ts' />
+
+// @strict: true
+
+//// class T {
+////     a: string;
+////
+////     static b: string;
+////
+////     private c: string;
+////
+////     d: number | undefined;
+////
+////     e: number | string;
+////
+////     foo: T;
+//// }
+
+verify.codeFixAvailable();
```

Comment:
Also test the content of the code fix -- use `verify.codeFix` and `verify.codeFixAll` (in separate tests).

Reply:
updated